lib/iinChecker.js

```diff
@@ -46,13 +50,7 @@
     this.options.messages = require( './i18n/' + this.options.language ); // ISO 639‑1
 
     this.isCacheable = function( cardInfo ) {
-      if ( this.options.cache &&
-          cardInfo.brand && cardInfo.brand !== this.brands.UNKNOWN &&
-          cardInfo.type && cardInfo.type !== this.nullValue
-      ) {
-        return true;
-      }
-      return false;
+      return (this.options.cache && cardInfo.brand && cardInfo.brand !== this.brands.UNKNOWN && cardInfo.type && cardInfo.type !== this.nullValue);
```

Comment:
I like this better than what was before but it's not that readable. One suggestion I have is to make it completely verbose and do something like this, totally up to you though
```
function checkCard( cardInfo ) {
	var self = this;
	// no cache so not cacheable
	if( !self.options.cache ) {
		return false;
	}
	// don't cache cards we don't know the brand for
	if( !cardInfo.brand || cardInfo.brand === self.brands.UNKNOWN ) {
		return false;
	}

	// don't cache cards we don't know the type of
	if( !cardInfo.type || cardInfo.type === this.nullValue ) {
		return false;
	}

	// cacheable!
	return true;

}
```